react/react_intro/README.md

```diff
@@ -350,28 +362,19 @@ function App() {
 }
 ```
 
-`App.css`
+After applying that `className`, we can create an `App.css` file:
+
 ```css
 .App-Div {
   display: flex;
   border: 8px solid;
 }
 ```
 
-Add some borders and margins around the other components as well to see how it's laid out.  Then you should see the following screen:
-
-![styledApp](./images/styledApp.png)
+And `import` it into `App.js`.
 
+Add some borders and margins around the other components to give you a better impression of how the components are laid out. After we're done, the app should look something like this:
 
```

Comment:
I think we can cut the content on adding flex, etc. from the README and/or recast it as an exercise that fellows do on their own in groups.